src/localize-router.config.ts

```diff
@@ -1,16 +1,16 @@
-import { Inject, OpaqueToken, Provider } from '@angular/core';
+import {Inject, InjectionToken, Provider} from '@angular/core';
 
 /**
  * Guard to make sure we have single initialization of forRoot
- * @type {OpaqueToken<LocalizeRouterModule>}
+ * @type {InjectionToken<LocalizeRouterModule>}
  */
-export const LOCALIZE_ROUTER_FORROOT_GUARD = new OpaqueToken('LOCALIZE_ROUTER_FORROOT_GUARD');
+export const LOCALIZE_ROUTER_FORROOT_GUARD = new InjectionToken('LOCALIZE_ROUTER_FORROOT_GUARD');
```

Comment:
`InjectionToken` is generic, so consider specifying the type: `InjectionToken<LocalizeRouterModule>`. Same for the rest of the tokens :)

Reply:
You are right! I will have to create a new PR though, that one is already merged